### ISSUE TITLE
Plasma Discover does not notify about pending updates after latest update

### ISSUE THREAD

**Issue by azioga** [PackageKit/PackageKit-Qt#41]
after latest update of libpackagekitqt5-1 DiscoverNotifier in plasma-kde stops notify about pending updates

reverting to previous version 1.0.2-1 it works

https://bugs.kde.org/show_bug.cgi?id=462170#c12
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1027831

who is the culprit? libpackagekitqt regression or Discover not having adjusted to handle
plural signals from PK properly ?
Best, Luca.

**Comment by ximion** [comment]
There's a fairly good chance that the cause of this issue is issue https://github.com/PackageKit/PackageKit-Qt/issues/42

**Comment by azioga** [comment]
> There's a fairly good chance that the cause of this issue is issue #42

I hope it, but I have not kded crashes

**Comment by ximion** [comment]
Are you sure? kded can auto-restart once it crashes... Unfortunately I can't reproduce the kded crash either, and `coredumpctl` doesn't show any issues on my system, so this may indeed be a separate bug.
There's a good chance that @aleixpol has an idea here :-)

**Comment by azioga** [comment]
sure. coredumpctl doesn't show any issues on my system too.

**Comment by aleixpol** [comment]
@azioga do you have DiscoverNotifier crashing?

The issue #42 refers to a kded crash coming from [apper](https://invent.kde.org/system/apper), we should be seeing the symptom elsewhere.

It would be interesting to run pkmon on an affected system and see if there's signals that are not waking DiscoverNotifier up for some reason.

**Comment by azioga** [comment]
Hi!
DiscoverNotifier never crashes. 
I have removed apper long time ago.

Here is pkmon after apt-get update

```
$ pkmon
Transactions:
 [none]
daemon connected=1
network status=online
updates-changed
Transactions:
 1      /18553_baacecee
/18553_baacecee allow_cancel 1
/18553_baacecee percentage   -1
/18553_baacecee role         get-updates
/18553_baacecee sender       /usr/lib/x86_64-linux-gnu/libexec/DiscoverNotifier
/18553_baacecee status       setup
/18553_baacecee status       loading-cache
/18553_baacecee percentage   50
/18553_baacecee percentage   100
/18553_baacecee status       query
/18553_baacecee status       loading-cache
/18553_baacecee percentage   10
/18553_baacecee percentage   100
/18553_baacecee status       install
/18553_baacecee package-id   libappimage1.0abi1;1.0.4-4;amd64;+auto:debian-testing-main
/18553_baacecee package      installing:libappimage1.0abi1;1.0.4-4;amd64;+auto:debian-testing-main:Core library for appimage
/18553_baacecee package-id   libpython3.11;3.11.1-2;amd64;+auto:debian-testing-main
/18553_baacecee package      installing:libpython3.11;3.11.1-2;amd64;+auto:debian-testing-main:Shared Python runtime library (version 3.11)
/18553_baacecee package-id   libsquashfuse0;0.1.105-1;amd64;+auto:debian-testing-main
/18553_baacecee package      installing:libsquashfuse0;0.1.105-1;amd64;+auto:debian-testing-main:FUSE filesystem to mount squashfs archives library
/18553_baacecee package-id   libxdgutilsbasedir1.0.1;1.0.1-3.1;amd64;+auto:debian-testing-main
/18553_baacecee package      installing:libxdgutilsbasedir1.0.1;1.0.1-3.1;amd64;+auto:debian-testing-main:Implementation BaseDir in Free Desktop Standards for C++
/18553_baacecee package-id   libxdgutilsdesktopentry1.0.1;1.0.1-3.1;amd64;+auto:debian-testing-main
/18553_baacecee package      installing:libxdgutilsdesktopentry1.0.1;1.0.1-3.1;amd64;+auto:debian-testing-main:Implementation Desktop Entry in Free Desktop Standards for C++
/18553_baacecee status       finished
/18553_baacecee exit code: success
Transactions:
 [none]
```
and this row in journal
`gen 05 13:46:50 luca DiscoverNotifier[2337]: packagekitqt.transaction: Unknown Transaction property: "Sender" QVariant(QString, ":1.71")`

**Comment by aleixpol** [comment]
Still not sure what could be the culprit. Discover is not using the sender at the moment.

**Comment by ximion** [comment]
This issue is also fixed by #48 
I'll create a bugfix release tomorrow.
